### ISSUE TITLE
Maintainers wanted!

### ISSUE THREAD

**Issue by edubarr** [edubarr/header-decor#84]
## TL;DR
Let me know if you'd like to take over as one of the maintainers of this library. If no one wants the responsibility then the repo and library will be marked as `Deprecated`.

## Help Wanted
I haven't had the time or inclination to maintain, update or improve this library in quite a long time. @starkej2 has been an invaluable help to me, as he has taken over most of the work looking at issues and cleaning things up. I consider it to be feature complete for what was my own use, but there are quite a few requests for fixes, features and improvements which I'll probably never implement.

This is something I've been thinking about (and postponing) for quite a while. I would like for users of the library to either know that it's being maintained or be warned that they are on their own when using this code. These two options translate into either finding people who are interested in taking over this repo _or_ deprecating this library.

If you'd like to take over then please let me know. I can create an organization and move the repo over. We could also switch to using [jitpack](https://jitpack.io) instead of the existing maven repo, which would give quite a bit of flexibility. I would still check how things are going every once in a blue moon and, who knows, might even contribute to it again!

If no one wants to take over then I'll mark the library as deprecated. The code and existing artifacts will remain in their respective places and everything _should_ work until it doesn't (incompatible support libraries, API changes, etc). 

Thanks,
Eduardo

**Comment by starkej2** [comment]
I'd be willing to help out. Hopefully others are interested in contributing too since I can't dedicate _too_ much time.

**Comment by Pankaj-Baranwal** [comment]
I don't know if the repo has been taken up for further maintenance yet.
So, if the "help wanted" label is still active, I would be happy dedicate my time to get the repo up to speed with the latest changes in Android as well as take care of the issues other users face.
Thanks.

**Comment by oshamahue** [comment]
I can dedicate some time too.

**Comment by Rafiqgithub** [comment]
I will help

**Comment by griffins** [comment]
i will help too

**Comment by hemimotors** [comment]
i will help too
